packages/story-editor/src/components/floatingMenu/elements/borderWidthAndColor.js

```diff
@@ -134,8 +134,11 @@ function BorderWidthAndColor() {
             value={border.color || BLACK}
             onChange={handleColorChange}
             hasInputs={false}
-            hasEyeDropper={false}
+            hasEyedropper={false}
             allowsOpacity={canHaveBorderOpacity}
+            allowsGradient={false}
+            pickerHasEyedropper={false} // override shared floating menu Color component TODO https://github.com/GoogleForCreators/web-stories-wp/issues/11024
+            allowsSavedColors={false}
```

Comment:
same set of updates as found here: https://github.com/GoogleForCreators/web-stories-wp/pull/11025/files#diff-3e101509bd8aebce46f213f43dc1e13358fabc6f9cccef75d7d4cce88f8de35c

Reply:
Also same as #11075 it seems?

Reply:
that one actually reenables the color picker, i noticed that my hotfix to avoid crashing was gone when i rebased that PR. Felipe identified an (unrelated) issue where the eyerdropper in the sidepanel outside of the color picker is out of sync with updating the canvas which took that PR back into in progress, so it's not going to make the release. Whereas this is just eliminating saved colors and eyedropper from floating menu color picker in the mean time.